### PR TITLE
samples: nrf_desktop: Profiled events selection

### PR DESCRIPTION
**Pull request by MarekPieta**
Change enables dynamic selection of profiled events using shell.

### CODE REVIEW

File: subsys/profiler/profiler_common_shell.c [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+			evt_mask |= BIT(i);
+		}
+
+		shell_fprintf(shell,
+			      SHELL_NORMAL,
+			      "Profiling all events %s\n",
+			      enable ? "enabled":"disabled");
+	} else {
+		const size_t index_cnt = argc - 1;
+		long long int event_indexes[index_cnt];
```
Comment (pdunaj) [inline subsys/profiler/profiler_common_shell.c]: long long?

Comment (MarekPieta) [reply subsys/profiler/profiler_common_shell.c]: long int, i change

File: subsys/profiler/profiler_common_shell.c [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+	/* If no IDs specified, all registered events are affected */
+	if (argc == 1) {
+		for (int i = 0; i < profiler_num_events; i++) {
+			evt_mask |= BIT(i);
+		}
+
+		shell_fprintf(shell,
+			      SHELL_NORMAL,
+			      "Profiling all events %s\n",
```
Comment (Qbicz) [inline subsys/profiler/profiler_common_shell.c]: `"Profiling all events %sabled\n", enable ? "en" : "dis"`

I would change it not only to save flash, but also to keep convention from other modules.

Comment (MarekPieta) [reply subsys/profiler/profiler_common_shell.c]: done

**Comment by MarekPieta** [comment]
@carlescufi, could you merge ?

**Pull request merged**